src/services/config-manager.ts

```diff
@@ -3,17 +3,17 @@ import { ConfigManager } from '../types/config-manager';
 
 // Encapsulates a Configuration
 export class ConfigManagerInstance implements ConfigManager {
-  private config: Map<string, string>;
+  private config: Map<string, any>;
```

Comment:
Interesting, but perhaps this could be `Map<string, string | boolean>` instead to make it a more explicit?

Reply:
Which has caused compiler errors 😄 I'll go back to the TS docs on this one.